tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -2207,13 +2208,9 @@ describe('Filing History List - with documents', () => {
     // init store
     filingHistoryListStore.setFilings([FILING_WITH_DOCUMENTS_LINK] as any)
 
-    // mock "get documents"
-    sinon.stub(axios, 'get').withArgs('businesses/CP0001191/filings/111/documents')
-      .returns(new Promise(resolve => resolve({
-        data: {
-          documents: {}
-        }
-      })))
+    // mock "fetch documents"
+    vi.spyOn((LegalServices as any), 'fetchDocuments').mockImplementation(
+      () => Promise.resolve({}))
```

Comment:
The old code returns the data from the network level, while the new code returns a typed object of documents from the app level.